EnglishFilter/sections/replace.txt

```diff
@@ -24,6 +24,8 @@
 ||ssaiplayback.*.boltdns.net/playback/once/v*/hls/*.m3u8$replace=/(#EXT-X-TARGETDURATION:[\S]*?)\n[\s\S]*?#EXT-X-DISCONTINUITY/\$1/,domain=news.com.au
 ! pornhub.com - preroll
 ||pornhub.com/view_video.php?viewkey=$replace=/vastXml/_vastXml/,important
+||pornhub.org/view_video.php?viewkey=$replace=/vastXml/_vastXml/,important
+||pornhub.net/view_video.php?viewkey=$replace=/vastXml/_vastXml/,important
```

Comment:
Better:
||pornhub.*/view_video.php?viewkey=$replace=/vastXml/_vastXml/,important,~third-party

Reply:
I agree with that fix suggestion, and have implemented it now.